runtime/node/src/shared/composerBot.ts

```diff
@@ -55,9 +68,9 @@ export class ComposerBot extends ActivityHandler {
     const rootDialog = this.dialogManager.rootDialog as AdaptiveDialog;
     const claimIdentity = turnContext.turnState.get(turnContext.adapter.BotIdentityKey);
     if (claimIdentity && SkillValidation.isSkillClaim(claimIdentity.claims)) {
-      rootDialog.autoEndDialog = true;
+      rootDialog.autoEndDialog = new BoolExpression(true);
     } else {
-      rootDialog.autoEndDialog = false;
+      rootDialog.autoEndDialog = new BoolExpression(false);

```

Comment:
We don't have the else branch in C# runtime now, let's remove it.

Reply:
OK, removed this branch.